requirements-dev.txt

```diff
@@ -9,7 +9,7 @@ mlflow==0.9.1
 # Documentation build
 sphinx==2.0.1
 nbsphinx
-numpydoc
+numpydoc<=0.8
```

Comment:
BTW, this version was set before but unset due to Python 3.5 support .. I should check Travis CI picks Python 3.5 correctly..

Reply:
what if you just pin it to 0.8?

Reply:
it upgrades Python version from 3.5 to 3.6. So it setting to 0.8.0 was removed.. which was .. here https://github.com/databricks/koalas/commit/47a36605904771f545ef95091c5483743761294b

Reply:
Seems like some numpydoc version higher than 0.8.0 on conda doesn't support Python 3.5.

If we pin both, it fails. I tested this condition before when we do mlflow.

Reply:
ah but we now use pip for python 3.5 installation. seems we can set the version